build/media-suite_helper.sh

```diff
@@ -1424,12 +1424,17 @@ do_unhide_all_sharedlibs() {
 }
 
 do_pacman_install() {
-    local installed
-    local pkg
-    installed="$(pacman -Qqe | /usr/bin/grep "^${MINGW_PACKAGE_PREFIX}-")"
+    local pkg msyspackage
+    while [ -n "$*" ]; do
+        case "$1" in
+        -m) msyspackage=y ;;
+        *) break ;;
+        esac
+    done
     for pkg; do
-        [[ "$pkg" != "${MINGW_PACKAGE_PREFIX}-"* ]] && pkg="${MINGW_PACKAGE_PREFIX}-${pkg}"
-        /usr/bin/grep -q "^${pkg}$" <(echo "$installed") && continue
+        [[ $msyspackage != "y" && "$pkg" != "${MINGW_PACKAGE_PREFIX}-"* ]] &&
+            pkg="${MINGW_PACKAGE_PREFIX}-${pkg}"
+        pacman -Qqe "$pkg" >/dev/null 2>&1 && continue
```

Comment:
If you remove the ^$ anchors this will match with packages that include the name you're looking for.
i.e. do_pacman_install python will match `python3` and `mingw-w64-x86_64-python`, `mingw-w64-x86_64-python3-docutils`, etc.